src/DotNetWorker/Definition/GrpcBindingMetadata.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    public class GrpcBindingMetadata : BindingMetadata
+    {
+        public GrpcBindingMetadata(BindingInfo bindingInfo)
+        {
+            Type = bindingInfo.Type;
+            Direction = bindingInfo.Direction == BindingInfo.Types.Direction.In ? Direction.In : Direction.Out;
```

Comment:
Why this logic here as opposed to assigning the direction value? Do we expect this would potentially carry something invalid?

Reply:
`BindingInfo.Types.Direction` is the Grpc type for the direction. Whereas, `Direction` here (renaming to `BindingDirection`) is our internal type. I didn't want to reuse the grpc type everywhere.